### DISCUSSION TITLE
What's Your Creative Leaning? Share Your Game Development Inspirations!

### DISCUSSION THREAD

**Discussion by dimensionscape** [haxecollab/ludum-dare-56#2] [General]
Hey team! With Ludum Dare 56 on the horizon, it's the perfect time to get to know each other's creative inclinations and game development inspirations. Every developer has unique preferences when it comes to making games—whether it's a specific genre, art style, storytelling approach, or game mechanic.

Let’s use this space to discuss what kind of games you enjoy making, the types of games you feel drawn to creatively, and what sparks your imagination in game design. Here are a few questions to get the conversation rolling:

    What genres do you prefer working on (platformers, RPGs, puzzles, etc.)?
    Do you enjoy focusing more on the art, story, mechanics, or something else?
    Are there any game developers, studios, or specific games that inspire you?
    What game features or mechanics are you excited to experiment with during Ludum Dare 56?

Feel free to share any past projects or examples of games you’ve enjoyed creating or that have influenced your approach to development. Let’s spark some inspiration and see how our combined creative leanings can influence our team’s direction for this jam!

**Comment by dimensionscape** [comment]
Ok, I'll go first!

I’ve always been drawn to story-driven games, especially those that blend tactical elements with deep, well-crafted narratives. Growing up with RPGs—including tabletop ones—games like Final Fantasy Tactics hold a special place in my heart.

In terms of development, I’m comfortable with most genres, and I particularly enjoy creating interfaces, systems, and mechanics, or focusing on storytelling. What excites me most about this jam is the chance to create an open-source entry alongside a community I share a passion with, using technology I love, while getting to know and collaborate with fellow open-source enthusiasts.

With over 20 years of experience in game development, I’ve worked on a variety of titles. I got an early start with multiplayer games in grade school, building my first 2D MMO engine in 2001 using Shockwave and Shockwave Multiuser Server. Between 2006 and 2014, I focused primarily on single-player games, developing sims, platformers, puzzles, and tower defense titles using Flash.

For the jam, I can contribute in several areas, including:

- Simple art
- Programming (any)
- UI/UX design
- Organization/Management
- Theme development
- Problem Solving

I look forward to bringing my skills and experience to this collaborative project

**Comment by ElastedAlorian** [comment]
Going second, I like fi- platformers and some RPGs. I'd say I would usually focus on everything but the music when it comes to making anything (since music is my weak spot)
For games that inspire me I'd say like any Mario, Sonic, WF, or cool Newgrounds indie game I find laying around. (Partial joke statement.)
:sunglasses:
